### DISCUSSION TITLE
bug Date Fr ( dd/mm/yyyy)

### DISCUSSION THREAD

**Discussion by KsentiniAhmed** [NG-ZORRO/ng-zorro-antd#6012] [General] [answered]
![image](https://user-images.githubusercontent.com/21038043/98183015-87e38880-1f07-11eb-97a6-efea11d56228.png)

when I write the date without selected it will not be automatically selected despite the date it is good:
Note: its working for some test like: 12/12/2009
while its not working by for 02/21/2019

**Comment by wenqi73** [comment, marked as answer]
Please import date-fns. See https://ng.ant.design/docs/i18n/en#how-to-format-a-date-using-date-fns

**Reply by KsentiniAhmed** [reply]
it doesn't work

**Reply by wenqi73** [reply]
Could you provide the stackblitz url?
